presto-product-tests/src/main/resources/tempto-configuration.yaml

```diff
@@ -71,6 +71,13 @@ databases:
     jdbc_pooling: true
     table_manager_type: jdbc
 
+  cassandra:
+    host: cassandra
```

Comment:
We should probably document the default hosts for hadoop, presto, mysql, postgres, and cassandra in the presto-product-tests readme (since currently, if you run it without ``cassandra`` in /etc/hosts, it'll fail in a not-necessarily-obvious way).